spec/email_notification_spec.rb

```diff
@@ -41,11 +41,34 @@ def post_notification
     end
 
     describe 'from an authorised email address' do
+      let(:notify_email_url) { 'https://api.notifications.service.gov.uk/v2/notifications/email' }
+      let(:notify_api_key) { 'mock_key-00000000-0000-0000-0000-000000000000-00000000-0000-0000-0000-000000000000' }
+      let(:notify_template_id) { '00000000-0000-4321-1234-000000000000' }
+      let!(:notify_stub) { stub_request(:post, notify_email_url).to_return(status: 200, body: {}.to_json) }
       let(:from_address) { 'adrian@adrian.gov.uk' }
+      let(:username) { 'MockUsername' }
+      let(:password) { 'MockPassword' }
 
-      it 'calls User#generate' do
-        expect_any_instance_of(User).to receive(:generate).with(email: from_address)
+      before do
+        ENV['NOTIFY_API_KEY'] = notify_api_key
+        ENV['NOTIFY_USER_SIGNUP_EMAIL_TEMPLATE_ID'] = notify_template_id
+
+        expect_any_instance_of(User).to \
```

Comment:
do you need \ in ruby? it's been a while

Reply:
Because this statement is over several lines without the `\` ruby will assume that you are calling `.to` without any arguments.